service/src/main/java/org/sunbird/util/user/UserExtendedProfileSchemaValidator.java

```diff
@@ -0,0 +1,79 @@
+package org.sunbird.util.user;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.everit.json.schema.Schema;
+import org.everit.json.schema.ValidationException;
+import org.everit.json.schema.loader.SchemaLoader;
+import org.json.JSONObject;
+import org.sunbird.exception.ProjectCommonException;
+import org.sunbird.exception.ResponseCode;
+import org.sunbird.keys.JsonKey;
+import org.sunbird.logging.LoggerUtil;
+import org.sunbird.util.DataCacheHandler;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class UserExtendedProfileSchemaValidator {
+    private static final String PRIMARY_EMAIL_FIELD = "primaryEmail";
+
+    private static LoggerUtil logger = new LoggerUtil(UserExtendedProfileSchemaValidator.class);
+    private static Map<String, String> schemas = new HashMap<>();
+
+    public static void loadSchemas() {
+        ObjectMapper mapper = new ObjectMapper();
+        try {
+            String schemaConfig = DataCacheHandler.getConfigSettings().get(JsonKey.EXTENDED_PROFILE_SCHEMA_CONFIG);
+            TypeReference<HashMap<String, Object>> typeRef = new TypeReference<HashMap<String, Object>>() {};
+            Map<String, Object> schemaMap = mapper.readValue(schemaConfig, typeRef);
+            for (Map.Entry entry : schemaMap.entrySet()) {
+                schemas.put(entry.getKey().toString(), mapper.writeValueAsString(entry.getValue()));
+            }
+        } catch (Exception e) {
+            logger.error("UserExtendedProfileSchemaValidator.loadSchemas :: failed to load schemas.", e);
```

Comment:
try getting RequestContext object for logging

Reply:
will keep this as a backlog as this change required in many places